src/modules/faqs/faqs.service.ts

```diff
@@ -13,9 +13,8 @@ export class FaqsService {
   ) {}
 
   async createFaq(createFaqs: CreateFaqDto, userId: string) {
-    const user = await this.userRepo.findOneBy({ id: userId });
-
     try {
+      const user = await this.userRepo.findOneBy({ id: userId });
       const existingFaq = await this.faqsRepo.findOneBy({ question: createFaqs.question });
```

Comment:
no catch block?

checkout the getUserRecord in the userService file. It handles this check.

Reply:
There is a catch block, boss